tests/test_compat_matrix.py

```diff
@@ -18,12 +18,10 @@
 
 from flict.flictlib.compat_matrix import CompatibilityMatrix, CompatMatrixStatus
 from flict.var import VAR_DIR
-
-MATRIX_FILE=os.path.join(VAR_DIR, "osadl-matrix.csv")
-
+from osadl_matrix import OSADL_MATRIX
```

Comment:
I am surprised that flake8 is happy with this. I would have expected more newlines here.

Reply:
Well, I'm not sure if we might have this newline check disabled, but certainly I don't see the need to enforce it for the test-files... :thinking:... I think we don't enforce any flake8 findings for the files under tests...